templates/hello/animation-snippet.html

```diff
@@ -0,0 +1,274 @@
+<!-- Any instances of {{ snippet_id }} are replaced with the ID number for the
+     snippet using this template. -->
+<div class="snippet" id="snippet-{{ snippet_id }}">
+  <style>
+    /* All CSS should be preceeded by the ID selector for this individual
+       snippet. */
+    #snippet-{{ snippet_id }}-contents {
+      opacity: 0;
+      position: absolute;
+      bottom: -10px;
+      left: -50px;
+      width: 547px;
+      text-align: center;
+      color: #3c3c3c;
+      transition: opacity .4s ease-in .2s;
+    }
+    #snippet-{{ snippet_id }}-contents p {
+      margin: 1rem 0;
+    }
+    #snippet-{{ snippet_id }}-contents a {
+      background: #00b7e2;
+      color: #fff;
+      display: inline-block;
+      padding: .8rem 2rem;
+      border-radius: .2rem;
+      transition: background ease-in .2s;
+    }
+    #snippet-{{ snippet_id }}-contents a:hover {
+        background: #00d1ef;
+    }
+    #snippet-{{ snippet_id }}-contents a:active {
+        background: #009dd5;
+    }
+    .show-contents-{{ snippet_id }} #snippet-{{ snippet_id }}-contents {
+      opacity: 1;
+    }
+    div.switch-{{ snippet_id }}#brandLogo {
+      position: relative;
+      background-size: auto;
+      height: 192px;
+      width: 447px;
+      transition: background-position .3s ease-in-out;
+    }
+    div.show-contents-{{ snippet_id }}#brandLogo {
+      /* move finished animation up to make room for text and button */
+      background-position: 0 -50px;
+    }
+  </style>
+
+  <div id="snippet-{{ snippet_id }}-contents">
+    <p>{{ text|safe }}</p>
+    <a href="{{ fallbackUrl|safe }}" id="snippet-{{ snippet_id }}-button">
+      {{ buttonLabel|safe }}
+    </a>
+  </div>
+</div>
+
+<script>
+//<![CDATA[
+(function() {
+
+  var snippet = document.getElementById('snippet-{{ snippet_id }}');
+
+  snippet.addEventListener('show_snippet', function() {
+
+    var snippetContents = document.getElementById('snippet-{{ snippet_id }}-contents');
+    var contentDelay = 10000; // Delay before showing text/button
+
+    // Preload the new animation image from the CDN 
+    var animationImg = new Image();
+    animationImg.src = 'https://snippets.allizom.org/media/files/08f669b5-5e85-4b68-8025-d27f76913ad0.gif';
+ 
+    // Wait for animation to load before 
+    animationImg.addEventListener("load", function() {
+      var brandLogo = document.getElementById('brandLogo');
+      brandLogo.style.backgroundImage = 'url(' + animationImg.src + ')';
+      brandLogo.className += ' switch-{{ snippet_id }}';
+      brandLogo.appendChild(snippetContents);
+      setTimeout(function() {
+        brandLogo.className += ' show-contents-{{ snippet_id }}';
+      }, contentDelay);
+    });
+
+
+    // Begin UITour shinanigins
```

Comment:
Most of the code below was taken directly, or slightly modified, from this page: https://github.com/mozilla/snippets/blob/master/templates/hello/simple-snippet.html